File: app/src/main/java/com/m0n0l0c0/LinkedChronos/model/Chrono.java
package com.m0n0l0c0.LinkedChronos.model;

import android.os.CountDownTimer;

/**
 * Created by juanje on 22/05/16.
 */
public class Chrono {

    private long miliseconds;
    private String name;
    private CountDownTimer countDownTimer;

    public Chrono() {
    }

    public Chrono(long milisecond, String name, CountDownTimer countDownTimer) {
        this.miliseconds = milisecond;
        this.name = name;
        this.countDownTimer = countDownTimer;
    }

    public long getMiliseconds() {
        return miliseconds;
    }

    public void setMiliseconds(int miliseconds) {
        this.miliseconds = miliseconds;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public CountDownTimer getCountDownTimer() {
        return countDownTimer;
    }

    public void setCountDownTimer(CountDownTimer countDownTimer) {
        this.countDownTimer = countDownTimer;
    }
}

File: app/src/main/java/com/scallop/linkedchronos/model/Chrono.kt
package com.scallop.linkedchronos.model

import android.os.CountDownTimer

/**
 * Created by juanje on 22/05/16.
 */
class Chrono(milisecond: Long, name: String, countDownTimer: CountDownTimer) {

    var miliseconds: Long = milisecond
        private set
    var name: String? = name
    var countDownTimer: CountDownTimer? = countDownTimer

}
